catalyst/tools/registry.py

```diff
@@ -186,12 +186,11 @@ def add_from_module(self, module, prefix: Union[str, List[str]] = None) -> None:
             TypeError: if prefix is not a list or a string
         """
         factories = {
-            k: v for k, v in module.__dict__.items() if inspect.isclass(v) or inspect.isfunction(v)
+            k: v
+            for k, v in module.__dict__.items()
+            if inspect.isclass(v) or inspect.isfunction(v) or inspect.isbuiltin(v)
         }
 
-        # Filter by __all__ if present
-        names_to_add = getattr(module, "__all__", list(factories.keys()))
```

Comment:
I'd appreciate it if you can add `force_add_all`/`ignore_all` parameter instead of just removing this line

Reply:
Ok👌